C_sharp_exercises_2/C_sharp_exercises_2/Exercises2.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+//Exercise 2:
+//Write a method in C# to display numbers, multiplication of number with frequency and frequency of a number of giving array.
+namespace C_sharp_exercises_2
+{
+    public class Exercises2
+    {
+        public void countingFrequencyAndMultiplication()
+        {
+            IList<int> listNumbers = new List<int>() { };
+            Dictionary<int, int> NumbersCount = new Dictionary<int, int>();
+            Console.Write("Enter your Number: ");
+            int inputNUmber = int.Parse(Console.ReadLine());
+            for (int z = 0; z < inputNUmber; z++)
+            {
+                Console.Write($"Enter your  {z + 1} integer  number:");
+                var initialNumber = Console.ReadLine();
+                bool parseSuccess = int.TryParse(initialNumber, out int inputNumber);
+                if (parseSuccess)
+                {
+                    Console.WriteLine("Added in list ");
+                    listNumbers.Add(inputNumber);
+                }
+                else
+                {
+                    Console.WriteLine("wrong number type try again ");
+                    z--;
+                    continue;
+                }
+            }
+            for (int i = 0; i < listNumbers.Count; i++)
+            {
+                if (!NumbersCount.ContainsKey(listNumbers[i]))
```

Comment:
you used library for only keeping keys, its better to use List or array for this case

Reply:
when i will search in list this name it will be n(x) time but if i will search in dictionary and if i not mistaken it will be constant time